src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -310,6 +312,8 @@ object DockerPlugin extends AutoPlugin {
       daemonUserUid := Some("1001"),
       daemonGroup := "root",
       daemonGroupGid := Some("0"),
+      chownUser := None,
+      chownGroup := None,
```

Comment:
```suggestion
      dockerChownUser := None,
      dockerChownGroup := None,
```

See comment in the `Keys.scala`.

Also move up to the settings starting with `docker*`